people.md

```diff
@@ -5,10 +5,9 @@ permalink: /people/
 ---
 {%- for i in site.data.people -%}
 {%- if i.members.size > 0 -%}
-# {{ i.title }}
+## {{ i.title }}
```

Comment:
If you want to change the size of the text, use CSS. `##` inserts `<h2>`. As far as I could see, we weren't using `<h1>` anywhere so the top level header should be `<h1>` which is inserted by `#`

Reply:
Good point. Fixing this now